pkg/plugins/plugins.go

```diff
@@ -22,6 +29,124 @@ type PluginKey struct {
 	IsInternal     bool
 }
 
+type implementaion struct {
```

Comment:
Let's export this so that plugins can use the type when implementing the version command.